package.json

```diff
@@ -19,15 +19,16 @@
     "start": "npm run build && concurrently \"gulp watch\" \"npm:storybook\"",
     "watch": "gulp watch",
     "gh-pages-legacy": "export NODE_ENV=prod && npm run release && node gulp/gh-pages.js",
-    "gh-pages": "export NODE_ENV=prod && npm run build-storybook && node gulp/gh-pages.js",
+    "gh-pages": "export PUBLIC_PATH=https://qld-gov-au.github.io/qg-web-template && npm run build-storybook && node gulp/gh-pages.js && echo 'publishing complete.'",
```

Comment:
Could this be a cicd sub command so others can run gh-pages locally and still have it work?

Reply:
this has been changed in https://github.com/qld-gov-au/qg-web-template/pull/499
could you review these scrrips there?